lib/pages/ship_fitting/bloc/ship_fitting_bloc/bloc.dart

```diff
@@ -186,13 +186,17 @@ class ShipFittingBloc extends Bloc<ShipFittingEvent, ShipFittingState> {
         },
       ).toList();
     }
-
+    // Filter out non integratable rigs
+    initialItems = initialItems.where(
+            (item) => !_itemRepository.excludeFusionRigs.any((itemId) => item.id == itemId)
+    ).toList();
     emit(OpenRigIntegratorDrawer(
       rigIntegrator: event.rigIntegrator,
       topGroup: filteredGroup,
       initialItems: initialItems,
       slotIndex: event.slotIndex,
       fitting: fitting,
+      blacklistItems: _itemRepository.excludeFusionRigs
```

Comment:
If the items are being filtered out of the `initialItems` list - do we still need to pass them through?